gradle-jdks-settings/src/main/java/com/palantir/gradle/jdks/settings/ToolchainJdksSettingsPlugin.java

```diff
@@ -189,4 +201,39 @@ private static boolean isGradleVersionSupported() {
                         .compareTo(GradleVersion.version(GradleJdksEnablement.MINIMUM_SUPPORTED_GRADLE_VERSION))
                 >= 0;
     }
+
+    private static void runGradleJdkSetup(Path rootProjectDir) {
+        Path buildDirectory = rootProjectDir.resolve("build");
+        createDirectories(buildDirectory);
+        CommandRunner.runWithLogger(
+                new ProcessBuilder().command("./gradle/gradle-jdks-setup.sh").directory(rootProjectDir.toFile()),
+                ToolchainJdksSettingsPlugin::writeStdOutput,
+                ToolchainJdksSettingsPlugin::writeStdErr);
+    }
+
+    private static Void writeStdOutput(InputStream inputStream) {
```

Comment:
looks like this was copied from CaResource.  All the methods seem the same except what they do with logger.  Would it make sense to make CommandRunner non-static and put the methods in there, then pass it a logger when created?

Reply:
I am not sure that I can pass the logger/re-use the logic.

- the Logger is different in CaResources (it is an [interface](https://github.com/palantir/gradle-jdks/blob/develop/gradle-jdks-setup/src/main/java/com/palantir/gradle/jdks/setup/CaResources.java#L50) that can be either a `stdLogger` or a `gradle logger`). 
- The logger used here is a `gradle logger` but when I log something here (the output would be shown in Intellij), I want to always log with `.lifecycle` to make sure the logs always show up, whereas in `CaResources` the default will be `info log level`